src/distributions/normal.jl

```diff
@@ -479,7 +479,7 @@ function Base.:-(left::MultivariateNormalNaturalParameters, right::MultivariateN
 end
 
 function lognormalizer(η::UnivariateNormalNaturalParameters)
-    return η.weighted_mean^2 / (4 * η.minus_half_precision) + log(-2 * η.minus_half_precision) / 2
+    return -η.weighted_mean^2 / (4 * η.minus_half_precision) - log(-2 * η.minus_half_precision) / 2
 end
 
 function lognormalizer(η::MultivariateNormalNaturalParameters)
```

Comment:
perhaps using `chollogdet` in this function might be better performant (although this wasn't  part of the PR ;)

Reply:
@bartvanerp Do you mean for the `MultivariateNormalNaturalParameters`?

Reply:
Yess, if this `η.minus_half_precision` variable is a symmetric posdef matrix, then it is advantageous to first compute its Cholesky decomposition, which can then be used to both efficiently compute the `logdet` and the `\`.